campaignion_newsletters/src/Subscription.php

```diff
@@ -23,6 +28,14 @@ class Subscription extends \Drupal\little_helpers\DB\Model {
   protected static $values = array('fingerprint', 'updated', 'last_sync');
   protected static $serial = FALSE;
 
+  /**
+   * Construct a new instance from data found in the DB.
```

Comment:
Mh that should be more like "Get an instance form DB or create a new one." or something like that.